docs/components/set-relation-many-to-many.md

```diff
@@ -0,0 +1,30 @@
+---
+docId: 'setrelationmanytomany'
```

Comment:
which menu this tutorial located ? i can't found it

please attach screenshot that this tutorial page located @rahmatalmubarak